schema/spec.md

```diff
@@ -348,6 +357,25 @@ Versions for a schema named `com.example.telemetrydata`:
 }
 ```
 
+#### `validation`
+
+- Type: Boolean
+- Description: Indicates whether or not the server will validate the Resource's
+  document(s) against the specified `format` attribute. A value of `true`
+  indicates that the server MUST reject any request that would cause any
+  Version of this Resource to be invalid per the rules as defined by the
+  `format` specification. Note, this includes a request to set this attribute
+  to `true`.
+
+  A value of `false` indicates that the server MUST NOT do any validation.
```

Comment:
This is interesting. It feels to me like it would be useful to make "validate but don't prevent" an option - e.g. "The server mustn't reject requests causing an invalid version, but can report warnings". I don't know where it would report warnings, admittedly.

Reply:
As you said... how do you return warnings?

I guess one way to do it would be to have some kind of "status" attribute that's server controlled that would tell you whether or not the doc is valid. Then not just the current user would know but anyone checking the registry later too. Let's discuss because it does make things a bit more complex for users.